rules/integrations/kubernetes/credential_access_kubernetes_secrets_accessed.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2022/05/17"
+maturity = "production"
+updated_date = "2022/05/17"
+integration = "kubernetes"
```

Comment:
How long has this integration existed. We may need to add a `min_stack_version` to only backport this to stacks when this became available

Reply:
The manifest says

Reply:
@brokensound77 kubernetes integration manifest says:

```
release: ga
conditions:
  kibana.version: "^8.2.0"
```